txp/Pos/Txp/Toil/Utxo/Util.hs

```diff
@@ -33,3 +35,11 @@ utxoToStakes = foldl' putDistr mempty . M.toList
   where
     plusAt hm (key, val) = HM.insertWith unsafeAddCoin key val hm
     putDistr hm (_, toaux) = foldl' plusAt hm (txOutStake toaux)
+
+utxoToAddressCoinPairs :: Utxo -> [(Address, Coin)]
```

Comment:
1. I think this function should return `HashMap Address Coin`. It applies `HM.toList` in the end. If caller needs `HashMap`, they will apply `HM.fromList` to the result and it will lead to unnecessary conversion to and from `HashMap`. If caller needs `[(Address, Coin)]`, they can use `HM.toList` explicitly. And I can't say that the latter case will be much more popular.
2. It would be good to add a haddock comment.